types/layui-src/tslint.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "max-line-length": [false],
+        "no-trailing-whitespace": false,
+        "space-before-function-paren": false,
+        "only-arrow-functions": [false]
+    },
+    "space-before-function-paren": false
```

Comment:
These are only added because of non-formatted content in your tests. 
Can you please make that one to default to recommented content:
```json
{
    "extends": "dtslint/dt.json"
}
```
and just run prettier on your source files?
```bash
npx prettier --write types/layui-src
```

Reply:
If I put tslint.json Modify what you requested,
Then when running test, the following error will be reported
```
1> ERROR: 3:22    only-arrow-functions         non-arrow functions are forbidden
1> ERROR: 3:30    space-before-function-paren  Spaces before function parens are disallowed
```
我要怎么做才能解决问题。
我使用idea的pretty工具，它也会在括号前加入空格

Reply:
What can I do to solve the problem.

I use idea's pretty tool, which also places spaces before parentheses

Reply:
Is there a document or information address on how to write test documents  for layui-src-tests.ts

Reply:
I read the  [How can I contribute](https://github.com/DefinitelyTyped/DefinitelyTyped#how-can-i-contribute)carefully, and then the local test solved the problem. Thank you

Reply:
sorry, could not answer prompty - timeonze diff 😄